combat-trainer.lic

```diff
@@ -3530,13 +3544,17 @@ class GameState
   end
 
   def wield_whirlwind_offhand
+    return if weapon_skill.include?'Twohanded'
     return unless currently_whirlwinding
     @equipment_manager.wield_weapon_offhand(whirlwind_offhand_name)
   end
 
   def determine_whirlwind_action
-    if @use_weak_attacks && Flags['ct-accuracy-ready']
-      update_analyze_array(true) if @analyze_combo_array.empty?
+    if @flame_for_expertise == true && ( (DRStats.mana < @flame_inner_fire_threshold) || (DRSkill.getxp('Expertise') < 15) )
```

Comment:
I like this addition to choose to analyze flame to increase IF if you're below a threshold. I would like to take advantage of that outside of whirlwinding since I don't have that ability yet. Thoughts on moving that logic to the `update_analyze_array` method so that for any barb in combat, if they're below their desired IF then it'll do flame otherwise it'll select something else based on their skill level?